### ISSUE TITLE
Missing context type in import statement of generated typescript resolver

### ISSUE THREAD

**Issue by mdaouas** [dotansimha/graphql-code-generator#1521]
**Describe the bug**
Context type is missing in import statement of generated resolver with module#type syntax in configuration of contextType

**To Reproduce**

1. Use any schema
2. Add a a context type in configuration file
```yaml
# ...
generates:
  path/to/file.ts:
    config:
      contextType: ./context#MyContext
    plugins:
      - typescript-resolvers
```
3. Run the following command: `graphql-codegen` with configuration file

**Expected behavior**
Generated resolver should have an import statement like
```typescript
import { MyContext} from './context';
```
and getting the following (missing the type)
```typescript
import { } from './context';
```

**Comment by dotansimha** [comment]
Hi @mdaouas , thank you for reporting it. 
I created a fix for it: https://github.com/dotansimha/graphql-code-generator/pull/1522 
I'll merge and release a new version (1.0.2) once CI passes.

**Comment by ctrlplusb** [comment]
Awesome @dotansimha - thanks for the super quick turnaround. I was just coming to report the same thing. :)

**Comment by dotansimha** [comment]
Fix available in 1.0.2.
